### DISCUSSION TITLE
Editing Process

### DISCUSSION THREAD

**Discussion by MedouneSGB** [MedouneSGB/Waxtane#14] [Announcements]
### 1 - Edit Readme.md :
To Update the Readme you have to edit /templates/README.md.
The Github Action is using it to generate the final Readme with data added.
Here is the [Editable Readme](https://github.com/MedouneSGB/Waxtane/blob/main/templates/README.md)

### 2 - Edit Waxtane Translation List :
To edit the Translation list use the /data/waxtane.csv.
The Github Action will automatically update the Readme adding the updated Translation List to it.
Here is the [Editable List of Translations](https://github.com/MedouneSGB/Waxtane/blob/main/data/waxtane.csv)

**Comment by MedouneSGB** [comment]
Hey @ManuSquall check this !

**Reply by ManuSquall** [reply]
Ho thanks bro, didn't figure it out before!
